blocks/single-column-layout-block/layouts/single-column-regular/index.story.jsx

```diff
@@ -23,22 +23,50 @@ const Navigation = () => <div style={styles}>Navigation</div>;
 const Component = ({ children }) => <div style={styles}>{children}</div>;
 const Footer = () => <div style={styles}>Footer</div>;
 
-export const layoutWithOneChild = () => (
+export const oneChildInBody = () => (
 	<SingleColumnRegularLayout>
 		<Navigation />
 		<>
-			<Component>Main 1</Component>
+			<Component>Body 1</Component>
 		</>
 		<Footer />
 	</SingleColumnRegularLayout>
 );
 
-export const layoutWithTwoChildren = () => (
+export const childrenInBody = () => (
```

Comment:
Same as the `oneChildInBody` story - the footer is being displayed above

Reply:
Fixed by this change...
https://github.com/WPMedia/arc-themes-blocks/pull/1374#discussion_r893502610
... and setting the sections in the right order in Storybook. Thanks!